src/main/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -872,4 +873,8 @@ default public void reloadLocalSchema() {
      * Sets the hinted handoff throttle in kb per second, per delivery thread.
      */
     public boolean resumeBootstrap();
+    
+    public List<CompositeData> getSSTableInfo(String keyspace, String table);
+
+    public List<CompositeData> getSSTableInfo();
```

Comment:
I don't see anything on `nodetool` side that would call into these new methods. Will there be another pull request against that repository to enable this functionality or am I missing something obvious?

Reply:
Yes. https://github.com/scylladb/scylla-tools-java/pull/106

Reply:
Aah, thanks!